src/abi/interface.ts

```diff
@@ -10,6 +11,8 @@ export class ABIInterface {
   public readonly name: string;
   public readonly description?: string;
   public readonly methods: ABIMethod[];
+  /** [ARC-28](https://arc.algorand.foundation/ARCs/arc-0028) events that MAY be emitted by this contract */
+  public readonly events?: ARC28Event[];
```

Comment:
I don't see anything in the ARC-28 spec about events being associated with interfaces. I only them as a property of either Contracts or Methods, so this feels like it shouldn't be here